crates/gpui/src/platform/linux/wayland/client.rs

```diff
@@ -378,14 +388,24 @@ impl Dispatch<wl_keyboard::WlKeyboard, ()> for WaylandClientState {
                 let key = if matches!(
                     key_sym,
                     xkb::Keysym::BackSpace
+                        | xkb::Keysym::Escape
+                        | xkb::Keysym::Home
+                        | xkb::Keysym::End
+                        | xkb::Keysym::Delete
+                        | xkb::Keysym::Insert
+                        | xkb::Keysym::Menu
                         | xkb::Keysym::Left
                         | xkb::Keysym::Right
                         | xkb::Keysym::Down
                         | xkb::Keysym::Up
+                        | xkb::Keysym::Page_Down
+                        | xkb::Keysym::Page_Up
                         | xkb::Keysym::Super_L
                         | xkb::Keysym::Super_R
```

Comment:
`tab` and `space` are inputed as `\t` and `\s`, should they be in this list? What we really want is a `is_keysym_printable()` function, but it always ends up as a list.

There's missing keysyms here. We might want to use a list from somewhere that has already done this more thouroughly, e.g. [from chromium](https://chromium.googlesource.com/chromium/src/+/b1c094c6916ffdef2820948bbd7fb6ad24382b59/ui/events/keycodes/dom/dom_code_data.inc).

Reply:
I'm also looking into translating control characters into the key pressed, so that we could use ctrl-(key) keymaps. (https://www.geeksforgeeks.org/control-characters/)

Reply:
Isn't the code below doing the work that you're trying to do there?

See line 407-408 of this file post patch, right below.

Reply:
Yep, it does... My bad, and thank you, i'll close my PR